nexus-repository-r/src/main/java/org/sonatype/nexus/repository/r/internal/RProxyFacetImpl.java

```diff
@@ -124,76 +118,55 @@ public void setCacheInfo(final Content content, final CacheInfo cacheInfo) {
     tx.saveAsset(asset);
   }
 
-  private Content putArchive(final String path, final String filename, final Content content) throws IOException {
+  private Content putArchive(final String path, final Content content) throws IOException {
+    checkNotNull(path);
+    checkNotNull(content);
```

Comment:
do we need to check for null private and protected methods? 
There is no @nullable modifiers on any parameters

Reply:
done in 7a2327f930a2b894b69a6e3faae6338d3eadae6e